daemon/listener/listener.js

```diff
@@ -298,6 +316,13 @@ process.argv.forEach(arg => {
 })
 
 const config = {
-  webhook: args['--webhook']
+  // Call webhook to process event.
+  webhook: args['--webhook'],
+  // Index events in the search index.
+  search: args['--search'],
```

Comment:
"search" might be the wrong name for this - it sounds too much like an action. Like you want the listener to search something.

"elasticsearch" would be fine though.

Reply:
changed to elasticsearch